vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -485,6 +499,22 @@ public static boolean isSoloSide(String side) {
   }
 
 
+  /**
+   * @return True if this is currently a multiPlayer game (either connected to a server, or more than one player side allocated)
+   */
+  public boolean isMultiPlayer() {
+    if (GameModule.getGameModule().getServer().isConnected()) return true;
+
+    int allocatedSideCount = 0;
+    for (PlayerInfo p : players) {
+      ++allocatedSideCount; // NB. Intentionally not excluding observers.
+    }
```

Comment:
I just noticed this. So we're looping to count the items instead of using `players.size()`?

Reply:
Those last four lines could be `return players.size() > 1;`.